packages/mask/src/plugins/Trader/helpers/pair.ts

```diff
@@ -22,7 +28,9 @@ export function getPairAddress(factoryAddress: string, initCodeHash: string, tok
                 ...cache?.[token0Addr],
                 [token1Addr]: getCreate2Address(
                     factoryAddress,
-                    keccak256(['bytes'], [pack(['address', 'address'], [token0Addr, token1Addr])]),
+                    stable
+                        ? keccak256(['bytes'], [pack(['address', 'address', 'bool'], [token0Addr, token1Addr, stable])])
+                        : keccak256(['bytes'], [pack(['address', 'address'], [token0Addr, token1Addr])]),
```

Comment:
Could you explain why adding a `stable` parameter?

Reply:
Hi @brymut, could you please paste some source code here?

Reply:
Hi, please check the code for the deployed contract, `stable` bool required to obtain pair addresses in `pairFor` function https://ftmscan.com/address/0xa38cd27185a464914D3046f0AB9d43356B34829D#code